lib/imports/api/authority/server/authority.js

```diff
@@ -52,6 +60,38 @@ export class Authority {
   }
 
   /**
+  * Stores a client's connection. Allows to handle disconnect later.
+  * @param {Object}     params
+  * @param {Connection} params.connection   Meteor Connection object of the client
+  */
+  storeConnection ({ connection }) {
```

Comment:
each Authority stores each DDP connection it's given, and then listens for its `onClose` and then checks if any remaining clients are still connected